app/views/system_admin/funding_uploads/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render PageTitle::View.new(i18n_key: "funding_uploads.new.#{funding_type}", has_errors: @funding_upload_form.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: "Upload funding information", href: funding_uploads_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop" >
+    <%= register_form_with(model: @funding_upload_form, url: funding_uploads_path) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.hidden_field :funding_type, value: funding_type %>
+
+      <h1 class="govuk-heading-l"><%= t("components.page_titles.funding_uploads.new.#{funding_type}") %></h1>
+
+      <p class="govuk-body"><%= t(".#{funding_type}") %></p>
+
+      <ol class="govuk-list govuk-list--number">
+        <li>
+          <%= f.govuk_select :month,
+              options_for_select([["Choose month", nil]] + Date::MONTHNAMES.compact.map.with_index(1) { |month, index| [month, index] }, selected: f.object.month),
+              label: { text: "Select applicable month" },
+              hint: { text: "Please select the month you are uploading funding information for." } %>
+        </li>
+        <li>
+          <%= f.govuk_file_field(:file, label: { text: "Upload the CSV file" }, accept: ".csv") %>
```

Comment:
does `accept` 'accept' file extensions or do we need a mime type here?

Reply:
Yeah it'll only whitelist the listed filetypes